test/governance/candidacy.users.test.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * @group governance-candidacy
+ */
+
+import BN from "bn.js";
+import { GovernanceUser } from "../../utils/Framework/User/GovernanceUser";
+import { Keyring } from "@polkadot/api";
+import { Bank } from "../../utils/Framework/Supply/Bank";
+import { Node } from "../../utils/Framework/Node/Node";
+import { SudoUser } from "../../utils/Framework/User/SudoUser";
+import { UserFactory, Users } from "../../utils/Framework/User/UserFactory";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+import { getEnvironmentRequiredVars, waitForNBlocks } from "../../utils/utils";
+
+const { chainUri: environmentUri } = getEnvironmentRequiredVars();
+const termDuration = 50; // Set as a const in mangata-node
+
+let bootnode: Node;
+let keyring: Keyring;
+let sudo: SudoUser;
+
+let candidate: GovernanceUser;
+let voter: GovernanceUser;
+let bank: Bank;
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(15000000);
+
+beforeAll(async () => {
+  await cryptoWaitReady(); // Wait for Polkadots WASM backend
+
+  bootnode = new Node(environmentUri);
+  await bootnode.connect();
+  await bootnode.subscribeToHead();
+
+  keyring = new Keyring({ type: "sr25519" });
+  sudo = UserFactory.createUser(Users.SudoUser, keyring, bootnode) as SudoUser;
+});
+
+beforeEach(async () => {
+  // Don't run tests at the end of a term
+  if (termDuration % bootnode.lastBlock! >= 45) {
+    waitForNextTerm();
+  }
+
+  // Initialize actors
+  candidate = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+  voter = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+  bank = new Bank(sudo);
+
+  // Fund accounts
+  await voter.addMGATokens(bank.sudoUser, new BN(Math.pow(10, 17).toString()));
+  await candidate.addMGATokens(
+    bank.sudoUser,
+    new BN(Math.pow(10, 16).toString())
+  );
+
+  // Subscribe to events
+  candidate.node.subscribeToUserBalanceChanges(candidate);
+  voter.node.subscribeToUserBalanceChanges(voter);
+});
+
+describe("Governance -> Candidacy -> Users", () => {
```

Comment:
what do you prefer? 
`describe("Governance -> Candidacy -> Users"` 
or
`describe("As a Governance user I can: "`

I think I like your approach a bit more :+1: , even though it results on fixing some describes in the existing tests :)

Reply:
Your example is better if we're following BDD but as we aren't I think it's nicer to avoid the extra words :)